tests/Integration/Console/ConsoleApplicationTest.php

```diff
@@ -86,3 +100,19 @@ public function handle(Schedule $schedule)
         //
     }
 }
+
+class HyperlinkCommandStub extends Command
+{
+    protected $signature = 'foo:hyperlink';
+
+    public function handle()
+    {
+        if ($this->option('ansi')) {
```

Comment:
This should not happen within the command.

Reply:
The reason I'm doing this in the command is because the output is not instantiated until the command is ran here:
https://github.com/laravel/framework/blob/6.x/src/Illuminate/Foundation/Testing/PendingCommand.php#L133

An option would be to override the `initialize` method in the stub:
https://github.com/symfony/console/blob/4.3/Command/Command.php#L185

Otherwise I would need to mock the output myself inside the test and run it with `$this->withoutMockingConsoleOutput()`. And then I would call `Illuminate\Console\Application::call('foo:hyperlink')` and then `Illuminate\Console\Application::output()` to perform assertions on this.